CMakeLists.txt

```diff
@@ -21,7 +21,7 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 #####################################################################################
-cmake_minimum_required(VERSION 3.5)
+cmake_minimum_required(VERSION 3.15 FATAL_ERROR)
```

Comment:
bump up?  It was 3.5 now it is 3.15 ?

Reply:
I fell for the same initially. 
3.5 is not 3.50. 
3.15 > 3.5

Reply:
Fifteen > five